build.sh

```diff
@@ -40,6 +40,7 @@ usage()
     echo "-configureonly - do not perform any builds; just configure the build."
     echo "-skipconfigure - skip build configuration."
     echo "-skipnative - do not build native components."
+    echo "-skipmsbuild - skip building managed binaries."
```

Comment:
I would call it `skipmanaged`. I think skipmsbuild is not a good name for this.

Reply:
Fixed now. Thanks!